pkg/reconciler/revision/resources/queue.go

```diff
@@ -350,7 +350,7 @@ func makeQueueContainer(rev *v1.Revision, cfg *config.Config) (*corev1.Container
 			Value: cfg.Deployment.ConcurrencyStateEndpoint,
 		}, {
 			Name:  "CONCURRENCY_STATE_TOKEN_PATH",
-			Value: path.Join(concurrencyStateTokenVolumeMountPath, concurrencyStateTokenName),
+			Value: path.Join(queue.TokenVolumeMountPath, queue.ConcurrencyStateTokenFilename),
```

Comment:
Yes, we need to keep this. 

iirc, the reason for having separate constants for the path and token name was in case we added other tokens in the future